docs/stellarphot/user_guide/using_photometry_data.rst

```diff
@@ -1,15 +1,28 @@
 Working with  Photometry Data
 #############################
 
-Photometry objects always have these attributes, also available dictionary-style:
-
-+ ``mag`` -- the calibrated magnitude; may be missing (e.g. your data prior to calibration)
-+ ``mag_err`` -- error in ``mag``
-+ ``inst_mag`` -- the instrumental magnitude; may be missing (e.g. catalog data)
-+ ``inst_mag_err`` -- error int the ``inst_mag``
-+ ``band`` -- the filter of the magnitude; required
-+ ``BJD`` -- Barycentric Julian Date of the midpoint of the observation; may be missing/masked
-+ ``RA2000`` -- right ascension in degrees, in the ICRS frame at epoch 2000
-+ ``DEC2000`` -- declination in degrees in the ICRS frame at epoch 2000
-
-There may be additional fields.
+
+Photometry data attributes
+--------------------------
+
+`~stellarphot.PhotometryData` objects always have these attributes, also available dictionary-style:
+
++ ``coord`` -- an astropy `~astropy.coordinates.SkyCoord` object with the coordinates for each star
++ ``mag_inst`` -- the instrumental magnitude
++ ``mag_inst_error`` -- error in the instrument magnitude
```

Comment:
shouldn't it be "instrumental magnitude" instead of "instrument magnitude"?  Also, for each star, right?